pkg/metrics/metrics.go

```diff
@@ -0,0 +1,65 @@
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+var log = logf.Log.WithName("registration_metrics")
+var Reg *prometheus.Registry
+
+const (
+	ResponseMetricLabelApprove = "Approve"
+	ResponseMetricLabelReject  = "Reject"
```

Comment:
maybe "Rejected" would be better?

Reply:
done in [71cfda3](https://github.com/codeready-toolchain/registration-service/pull/342/commits/71cfda305b156d0e68897a111e1709258ef3b1fb)